tests/drivers/clock_control/stm32_clock_configuration/boards/b_u585i_iot02a_msis_24.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Warning: This overlay relies on initial nucleo_u575zi_q configuration
+ *          For clarity, nodes are over written instead of deleted
+ *          Any change to board configuration has impact on this file
+ */
+
+&clk_lse {
+	status = "okay";
+};
```

Comment:
LSE is necessary for MSI autocalibration (PLLmode).
But I guess it is possible to work on MSI without PLL mode, so  without LSE.
Could be worth to have a new overlay with MSI but without LSE ?

Reply:
Sure, aim of current PR was to introduce the test suite, with the target of getting more tests in when possible.
I'm letting this proposal for a new PR if need is felt.